public/wp-content/themes/coloredcow/src/scss/assets/bootstrap/_card.scss

```diff
@@ -43,7 +43,7 @@
 }
 
 .card-subtitle {
-  margin-top: -$card-spacer-y / 2;
+  margin-top: math.div(-$card-spacer-y, 2);
```

Comment:
did you manually update this file?

Reply:
Actually, Bootstrap version 5 contains the files having the previous version of implementation instead of using `math.div( $variables, number)`. I think as of now we need to take the manual route for the execution of Dart SASS. Let me know your thoughts